### PR TITLE
add docs on how to execute jmeter tests

### PR DESCRIPTION
**Pull request by IlijazM**

### CODE REVIEW

**Comment by delvh** [comment]
Please fix your branch. Why are there Git Card Game testplans here?

**Review by Gr33ndev — APPROVED** [review]
the test plans have no place here, but the documentation is good

File: dev-manuals/languages/jmeter/run-stress-tests.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Or open up JMeter > `file` > `open`
+
+## Run jmeter test
+
+Press on the start button in the toolbar.
+
+<img src="images/start.png" />
+
+To stop it press the obviouse `Stop` button.
```
Comment (delvh) [inline dev-manuals/languages/jmeter/run-stress-tests.md:25]:
```suggestion
To stop it press the obvious `Stop` button.
```

File: dev-manuals/languages/jmeter/run-stress-tests.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## See reports
```
Comment (delvh) [inline dev-manuals/languages/jmeter/run-stress-tests.md:27]:
````suggestion

### From the command line

Use
```bash
$JMETER_EXECUTABLE -t $LOAD_PROFILE -n $ARGUMENTS
```
to test a load profile without the UI. \

## See reports
````

File: dev-manuals/languages/jmeter/run-stress-tests.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+On the left navigate to `View Result Tree`
+
+<img src="images/report.png" />
+
+## Clear reports
+
+Press on one item in the tree and press `Ctrl + E`.
```
Comment (delvh) [inline dev-manuals/languages/jmeter/run-stress-tests.md:35]:
```suggestion
Click on one item in the tree and press <kbd>Ctrl<kbd> + <kbd>E<kbd>.
```

File: dev-manuals/languages/jmeter/run-stress-tests.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+<img src="images/start.png" />
+
+To stop it press the obviouse `Stop` button.
+
+## See reports
+
+On the left navigate to `View Result Tree`
+
+<img src="images/report.png" />
```
Comment (delvh) [inline dev-manuals/languages/jmeter/run-stress-tests.md:31]:
```suggestion
![seeing results example](images/report.png)
```

File: dev-manuals/languages/jmeter/run-stress-tests.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Press on the start button in the toolbar.
+
+<img src="images/start.png" />
+
+To stop it press the obviouse `Stop` button.
+
+## See reports
+
+On the left navigate to `View Result Tree`
```
Comment (delvh) [inline dev-manuals/languages/jmeter/run-stress-tests.md:29]:
```suggestion
Navigate to `View Result Tree` on the left.
```

**Review by delvh — APPROVED** [review]

**Pull request merged**